### PR TITLE
Fix installation/releasenotes for SLE12

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: tests/installation/releasenotes.pm [hunk 1]
```diff
@@ -27,7 +27,12 @@ sub run() {
         send_key "alt-e";    # open release notes window
     }
     else {
-        send_key "alt-l";    # open release notes window
+        if (check_var('VIDEOMODE', 'text')) {
+            send_key "alt-l";    # open release notes window
+        }
+        else {
+            assert_and_click('release-notes-button');
+        }
```
Comment (dzedro) [inline tests/installation/releasenotes.pm]: Why change simple alt-l working for text and graphical mode ?

Comment (dzedro) [inline tests/installation/releasenotes.pm]: I think I know the answer, but I guess that's the reason why was releasenotes test removed from released products

Comment (sysrich) [inline tests/installation/releasenotes.pm]: alt-l works in SLE 12 SP2 and SP1...this poor gentleman has to test SLE 12 GA...

Comment (dzedro) [inline tests/installation/releasenotes.pm]: I know, never ending story aka release notes :)

**Comment by coolo** [comment]
did you already answer why we need to test this for GA?

**Comment by sysrich** [comment]
@coolo I think I heard something about some QAM team lead wants it

**Comment by scarabeusiv** [comment]
We had regressions in the relnotes displaying so it is quite smart to test for it. Note we had this prob on sle11 not 12 but why not be ready.

**Comment by sysrich** [comment]
Needles MR #68 on Gitlab

**Pull request merged**